R/structural.properties.R

```diff
@@ -2150,6 +2155,13 @@ bfs <- function(
     }
   }
 
+  if (lifecycle::is_present(father)) {
+    lifecycle::deprecate_warn("2.0.4", "bfs(father)", "bfs(parent)")
```

Comment:
Surprisingly, this code works, but the following is a bit more correct because `parent` and `father` are argument names:

```suggestion
    lifecycle::deprecate_warn("2.0.4", "bfs(father = )", "bfs(parent = )")
```